/orbslam3_odometry: 
  ros__parameters: 
    topic_camera_left: /zed/zed_node/left/image_rect_color
    topic_camera_right: /zed/zed_node/right/image_rect_color
    topic_imu: /imu/data
    
    # For monocular version only. It indicates whether to subscribe to camera left or right topic for images. 
    #   true    -> camera left's topic 
    #   false   -> camera right
    is_camera_left: true

    # For monocular version only, since it has no depth info. It indicates the scale to multiply the calculated position (usually between 10 and 20)
    scale_position_mono: 15

    # For monocular version only. Degree to move the calculated pose. If 0 pose will not be changed.
    degree_move_pose_mono: 16

    # Topic for odometry pubblication
    topic_orbslam_odometry: /Odometry/orbSlamOdom
    topic_header_frame_id: os_track
    topic_child_frame_id: orbslam3

    # Visualizer of map created by ORBSLAM3 and images passed with ROS2. 
    #   false to disable the visualizer
    #   true to enable it
    pangolin_visualization: true

    # Full file path to vocabulary of ORB_SLAM3 
    path_vocabulary: src/orbslam3_odometry/config/ORBvoc.txt

    # Full file path to yaml settings (for ORB_SLAM3)
    path_yaml_settings: src/orbslam3_odometry/config/zed2-stereo.yaml

    # ORB_SLAM3 mode. Poxibilities are:
    #   mono
    #   stereo
    #   stereo_imu
    # Monocular mode will subscribe to topic_camera_left 
    # if topic_is_camera_left is true 
    system_mode: mono 